src/components/TileGallery/_tile-gallery.scss

```diff
@@ -0,0 +1,213 @@
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~carbon-components/scss/globals/scss/typography';
+
+// StatefulTileGallery extra content
+.extraContent {
```

Comment:
Please follow BEM naming convention for classes

.#{$prefix}--stateful-tile-gallery__extra-content {
  ...
}

Reply:
@tay1orjones wouldn't `extra-content` here be a block, i.e. `.stateful-tile-gallery--extra-content`?

Reply:
It depends on the styles. `extra-content` appears to be a nested element within the component-level block of `stateful-tile-gallery` (or maybe just `tile-gallery`?), but yes, a nested element can use a new block name as long as the styles are independent of the parent.

[Some additional info from Carbon's developer handbook](https://github.com/carbon-design-system/carbon/blob/master/docs/developer-handbook.md#start-a-new-block-or-element)

Reply:
Looking better! Still a few camel cased classnames and items not scoped under a block level identifier.

Reply:
still see a few camelCases as Taylor mentioned.